components/function-controller/internal/controllers/serverless/function_reconcile_test.go

```diff
@@ -1295,8 +1295,8 @@ func TestFunctionReconciler_Reconcile(t *testing.T) {
 	t.Run("should reconcile function with RuntimeImageOverride", func(t *testing.T) {
 		//GIVEN
 		g := gomega.NewGomegaWithT(t)
-		runtimeImageOverride := "any-custom-image"
-		inFunction := newFixFunctionWithCustomImage(testNamespace, "custom-runtime-image", "initial-custom-image", 1, 2)
+		runtimeImageOverride := "any-custom-fnImage"
```

Comment:
I would change all `fnImage` to `fn-image` to be consistent

Reply:
Changed, it was the unwanted child of fast word refactor